python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -9994,3 +9995,21 @@ def test_dataframe_duplicated(data, subset, keep):
     actual = gdf.duplicated(subset=subset, keep=keep)
 
     assert_eq(expected, actual)
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        {"col": [{"a": 1.1}, {"a": 2.1}, {"a": 10.0}, {"a": 11.2323}, None]},
+        {"a": [[{"b": 567}]] * 10},
```

Comment:
Might be good to add a `None` to this data too?

Reply:
Yup, done 👍